vm/rust/versioned_constants_13_0.json

```diff
@@ -137,6 +137,11 @@
             "syscall_base_gas_cost": 1
         },
         "keccak_round_cost_gas_cost": 180000,
+        "sha256_process_block_gas_cost": {
```

Comment:
I would have thought each version of the versioned_constants.json file would be fixed. Do you know if running blockifier v6 with this modified versioned constants.json produces the same results (ie is it backward compatible?)?

Reply:
No, it's not backward compatible. The reason is because `VersionedConstants` struct requires some new fields which were not present in old files